modules/nf-core/nacho/qc/tests/main.nf.test.snap

```diff
@@ -0,0 +1,44 @@
+{
+    "Salmon - RCC files": {
+        "content": [
+            [
+                
+            ],
+            [
+                
+            ],
+            [
+                
+            ],
+            [
```

Comment:
It is not snapshoting the file names

Reply:
@atrigila I was working on that! You may check now, I had to adjust the test a little bit and, in the meantime, added some extra checks